test/test-notification-model-updates.js

```diff
@@ -0,0 +1,382 @@
+var ServiceManager = require('../index').ServiceManager;
+var async = require('async');
+var meshServer = require('../index').meshServer;
+var os = require('os');
+var test = require('tap').test;
+
+test('Test notifications', function(t) {
+  var app = meshServer(new ServiceManager());
+
+  t.test('Initialize models', function(tt) {
+    var s = {
+      id: 1,
+      name: 's1',
+      _groups: [{id: 1, name: 'g1', scale: 1}]
+    };
+    var e = {
+      id: 1,
+      address: '127.0.0.1',
+      APIPort: 5000,
+      totalCapacity: 1,
+      remainingCapacity: 0
+    };
+    var i = {
+      executorId: 1,
+      serverServiceId: 1,
+      groupId: 1
+    };
+
+    var ServerService = app.models.ServerService;
+    var ServiceInstance = app.models.ServiceInstance;
+    var Executor = app.models.Executor;
+    async.series([
+      ServerService.create.bind(ServerService, s),
+      Executor.create.bind(Executor, e),
+      ServiceInstance.create.bind(ServiceInstance, i)
+    ], function(err) {
+      tt.ok(!err);
```

Comment:
This wasn't erroring out in one of my tests but I may have just had an old version of tap or assert.
I'll switch it out.